README.md

```diff
@@ -433,6 +438,56 @@ To test functionality try uploading file to allocation. You should get the follo
 Upload failed. this options for this file is not permitted for this allocation: 
 file_option_not_permitted.
 ```
+##### Add Blobber
+
+Use `add_blobber` flag with [update allocation](#update-allocation) command to add blobber to allocation.
+
+Here are the necessary parameters for adding blobber.
+
+| Parameter     | Description                                                 | Valid Values |
+| ------------- | ----------------------------------------------------------- | ------------ |
+| --allocation  | Provide Allocation ID for adding blobber to allocation      | string       |
+| --add_blobber | Provide Blobber ID to add. Can be fetched using [List blobbers](#list-blobbers). | string       |
+
+Sample Command:
+
+```
+./zbox updateallocation --allocation $ALLOC --add_blobber 98f14362f075caf467653044cf046eb9e8a5dfee88dc8b78cad1891748245003
+```
+
+Sample Response:
+
+```
+Allocation updated with txId : d853a82907453d37ed978b9fc1a55663be99bb351d18cca31068c0dc95566edd
+```
+
+**Note:** Files will automatically be uploaded,splitted and stored on added blobber. 
+**Note:** An allocation is already hosted on a set of blobbers. To find a blobber that is available to add you should exclude the current set of blobbers hosting your allocation by checking them via [Get Allocation Info](#get)command.
+
+##### Replace Blobber
+
```

Comment:
Please add: 

Sometimes, a blobber might be mallfunctioning or faulty or the blobber might be slow becuase its far from your geolocation, in such cases you might have to replace the blobber with a  new one